static/app/views/performance/mobile/usePlatformSelector.tsx

```diff
@@ -0,0 +1,45 @@
+import {useMemo} from 'react';
+
+import {decodeScalar} from 'sentry/utils/queryString';
+import {useLocation} from 'sentry/utils/useLocation';
+import usePageFilters from 'sentry/utils/usePageFilters';
+import useProjects from 'sentry/utils/useProjects';
+import {PlatformSelector} from 'sentry/views/performance/mobile/screenload/screens/platformSelector';
+import {isCrossPlatform} from 'sentry/views/performance/mobile/screenload/screens/utils';
+
+export const PLATFORM_LOCAL_STORAGE_KEY = 'mobile-performance-platform';
+export const PLATFORM_QUERY_PARAM = 'platform';
+export const DEFAULT_PLATFORM = 'Android';
+
+function usePlatformSelector() {
+  const {selection} = usePageFilters();
+  const {projects} = useProjects();
+  const location = useLocation();
+
+  const project = useMemo(() => {
+    if (selection.projects.length !== 1) {
+      return null;
+    }
+
+    return projects.find(p => p.id === String(selection.projects));
```

Comment:
```suggestion
    return projects.find(p => p.id === String(selection.projects)) ?? null;
```

nit: unless we handle `undefined` vs `null` project differently where `usePlatformSelector` is called, I feel like it might be cleaner if we only return either `undefined` or `null`.